components/App.js

```diff
@@ -218,7 +218,7 @@ export const onclickBindings = {
   'twoD-inventoryCardTradeBtn': e => {
     const { menu } = getState();
     menu.trade.visible = true;
-    menu.trade.selectedItem = e.name
+    menu.trade.selectedItem = parseInt(e.name);
```

Comment:
Should have `, 10` since the default of parseInt is crazy.

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/parseInt